Gemfile.lock

```diff
@@ -170,17 +172,19 @@ GEM
     unf (0.1.4)
       unf_ext
     unf_ext (0.0.7.7)
-    unicode-display_width (1.7.0)
-    uswds-jekyll (5.3.0)
+    unicode-display_width (2.3.0)
+    uswds-jekyll (5.4.0)
```

Comment:
I ran `bundle update` on this because it helped pull in the latest libv8 gems. Looking at the site locally, everything looked good, but I didn't do super in-depth regression testing.